### PR TITLE
Add requirements.txt

### PR DESCRIPTION
**Pull request by Crinibus**
Hey, just thought you could add a requirements.txt to easily install the requirements for this repo.
Makes it easy for a user to install all requirements with a single command (especially if there are multiple):
```
pip install -r requirements.txt
```

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -27,6 +27,11 @@ There are four choices:
 1. [python3](https://www.python.org/downloads/)
 2. [fpdf](https://pypi.org/project/fpdf/)
 
+When Python is installed run the following command to install requirements:
```
Comment (januschung) [inline README.md:30]: What about? "Install require package with the following command:"

File: README.md [hunk 2]
```diff
@@ -27,6 +27,11 @@ There are four choices:
 1. [python3](https://www.python.org/downloads/)
```
Comment (januschung) [inline README.md:27]: Can you replace the fpdf line with requirement for pip please?

**Review by januschung — CHANGES_REQUESTED** [review]
apart from the comments, thanks so much

File: README.md (hunk 2 above)
Comment (Crinibus) [reply README.md:27]: What do you mean by "requirements for pip"? pip comes with the Python-installer :)

File: README.md (hunk 1 above)
Comment (Crinibus) [reply README.md]: Sounds better, I have updated it :)

File: README.md (hunk 2 above)
Comment (januschung) [reply README.md:27]: ah....beat me for that I started with python 2.7 which requires manual install of pip.....sorry about that

Comment (januschung) [reply README.md:27]: Can you take out line 28 and make line 27 as just "[python3](https://www.python.org/downloads/)" please? Will merge it once it is done.

Comment (Crinibus) [reply README.md]: Ahh no worries. It's updated now :)

**Review by januschung — APPROVED** [review]
lgtm

**Pull request merged**